app/jobs/exercise_verifier_job.rb

```diff
@@ -20,28 +20,21 @@ def perform(exercise)
     send_exercise_to_sandbox(exercise)
   end
 
-  def create_stub_tar(exercise)
-    exercise.create_file('stubfile')
+  def create_tar_files(exercise)
+    exercise.create_submission
 
-    `cd Stub/ && tar -cpf ../StubPackage.tar * && cd ..`
-  end
-
-  def create_model_solution_tar(exercise)
-    exercise.create_file('model_solution_file')
-    exercise.create_file('testfile')
-
-    `cd ModelSolution/ && tar -cpf ../ModelSolutionPackage.tar * && cd ..`
+    `cd ./submission_generation/tmp/Submission_#{exercise.id}/model/ && tar -cpf ../../../packages/ModelSolutionPackage_#{exercise.id}.tar * && cd ../../../..`
+    `cd ./submission_generation/tmp/Submission_#{exercise.id}/stub/ && tar -cpf ../../../packages/StubPackage_#{exercise.id}.tar * && cd ../../../..`
```

Comment:
Why are you changing directories here? Can't you just include the relative path to the input files in the arguments?